code/modules/mining/equipment/explorer_gear.dm

```diff
@@ -158,6 +158,19 @@
 	item_state = "seva"
 	resistance_flags = FIRE_PROOF
 
+/obj/item/clothing/suit/hooded/explorer/seva/standard
+	hoodtype = /obj/item/clothing/head/hooded/explorer/seva/standard
+
+/obj/item/clothing/head/hooded/explorer/seva/standard
+
+/obj/item/clothing/suit/hooded/explorer/seva/standard/Initialize()
+	. = ..()
+	AddComponent(/datum/component/armor_plate)
+
+/obj/item/clothing/head/hooded/explorer/seva/standard/Initialize()
+	. = ..()
+	AddComponent(/datum/component/armor_plate)
```

Comment:
no the point of seva is that it's for high skill players that can dodge and don't want to deal with ash storms why are you giving it armor